.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Unrelated change
Please do not commit .pytest_cache
It seems you used `git add .`
Please keep in mind that these files are temporary files that should not be pushed
If you get stuck while amending your commit, consider asking your doubts on our [gitter](https://gitter.im/coala/coala-bears) channel :)